README.md

```diff
@@ -1,69 +1,81 @@
-# Dependency Injection Checker
+# Guice Checker
 
-A common problem when programming is TODO.
-This results in a run-time exception.
+A common challenge when building systems that use [Guice](https://github.com/google/guice) is dealing with dependencies and ensuring they are properly defined. Omitting or misconfiguring dependencies can lead to runtime errors, impacting the reliability and stability of code.
 
-The Dependency Injection Checker guarantees, at compile time, that your code will
-not suffer that run-time exception.
+- [Guice Static Analysis](https://stackoverflow.com/questions/63446847/detect-all-injection-points-in-java-guice-via-static-code-analysis)
+- [Bug in Modules](https://stackoverflow.com/questions/10686360/guice-how-to-print-a-modules-bindings)
+- [Detecting Unused Bindings](https://stackoverflow.com/questions/55230580/guice-detect-unused-bindings)
 
+The Guice Checker is a static analysis tool designed to address this issue by providing compile-time guarantees that your Guice dependency mappings are accurately defined.
 
 ## How to run the checker
 
-First, publish the checker to your local Maven repository by running
-`./gradlew publishToMavenLocal` in this repository.
+First, build the checker by running `./gradlew clean build` and then publish the checker to your local Maven repository by running `./gradlew publishToMavenLocal` in this repository.
 
 Then, if you use Gradle, add the following to the `build.gradle` file in
-the project you wish to type-check (using Maven is similar):
+the project you wish to type-check:
 
 ```
-repositories {
-    mavenLocal()
-    mavenCentral()
+plugins {
+  id 'org.checkerframework' version '0.6.28'
```

Comment:
Before merging, update this to the latest Checker Framework plugin version (0.6.34 as I'm writing)